src/pt/mediocretoons/src/eu/kanade/tachiyomi/extension/pt/mediocretoons/MediocreToonsDto.kt

```diff
@@ -0,0 +1,143 @@
+package eu.kanade.tachiyomi.extension.pt.mediocretoons
+
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import kotlinx.serialization.Serializable
+import org.jsoup.Jsoup
+
+@Serializable
+data class MediocrePaginationDto(
+    val currentPage: Int? = null,
+    val totalPages: Int = 0,
+    val totalItems: Int = 0,
+    val itemsPerPage: Int = 0,
+    val hasNextPage: Boolean = false,
+    val hasPreviousPage: Boolean = false,
+)
+
+@Serializable
+data class MediocreListDto<T>(
+    val data: T,
+    val pagination: MediocrePaginationDto? = null,
+)
+
+@Serializable
+data class MediocreTagDto(
+    val id: Int = 0,
+    val nome: String = "",
+)
+
+@Serializable
+data class MediocreFormatDto(
+    val id: Int = 0,
+    val nome: String = "",
+)
+
+@Serializable
+data class MediocreStatusDto(
+    val id: Int = 0,
+    val nome: String = "",
+)
+
+@Serializable
+data class MediocreChapterSimpleDto(
+    val id: Int = 0,
+    val nome: String = "",
+    val numero: Float? = null,
+    val imagem: String? = null,
+    val lancado_em: String? = null,
+    val criado_em: String? = null,
+    val descricao: String? = null,
+    val tem_paginas: Boolean = false,
+    val totallinks: Int = 0,
+    val lido: Boolean = false,
+    val views: Int = 0,
+)
+
+@Serializable
+data class MediocreMangaDto(
+    val id: Int = 0,
+    val nome: String = "",
```

Comment:
You can use the ```@SerialName``` to keep the fields in English.

```suggestion
   @SerialName("nome")
    val name: String = "",
```

Reply:
adding @SerialName